safe_transaction_service/safe_messages/serializers.py

```diff
@@ -62,7 +69,9 @@ def get_valid_owner_from_signatures(
 
 class SafeMessageSerializer(SafeMessageSignatureParserMixin, serializers.Serializer):
     message = serializers.JSONField()
-    safe_app_id = serializers.IntegerField(allow_null=True, default=None)
+    safe_app_id = serializers.IntegerField(
+        allow_null=True, default=None, validators=[positive_integer_validator]
+    )
```

Comment:
Maybe we can use `min_value=0` instead of a custom validator.

Wdyt?

Reply:
I think I'm still on holidays mode 🤦🏼‍♂️

Reply:
Updated